From c5a88b2c987bda11b102de9ef1fca08191672a36 Mon Sep 17 00:00:00 2001
From: Vitaly Kuznetsov <vkuznets@redhat.com>
Date: Tue, 17 Apr 2018 13:07:53 +0200
Subject: [PATCH 1/3] Revert "azure: Fix publishing of hostname"

RH-Author: Vitaly Kuznetsov <vkuznets@redhat.com>
Message-id: <20180417130754.12918-2-vkuznets@redhat.com>
Patchwork-id: 79658
O-Subject: [RHEL7.6/7.5.z cloud-init PATCH 1/2] Revert "azure: Fix publishing of hostname"
Bugzilla: 1568717
RH-Acked-by: Eduardo Otubo <otubo@redhat.com>
RH-Acked-by: Mohammed Gamal <mgamal@redhat.com>
RH-Acked-by: Cathy Avery <cavery@redhat.com>

Turns out the downstream-only fix doesn't work, revert it.

This reverts commit 7a2f4618ba79ff3f7c4c7d026e645647a3da25d3.

Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 cloudinit/sources/DataSourceAzure.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cloudinit/sources/DataSourceAzure.py b/cloudinit/sources/DataSourceAzure.py
index a0ebd2b..48a3e1d 100644
--- a/cloudinit/sources/DataSourceAzure.py
+++ b/cloudinit/sources/DataSourceAzure.py
@@ -27,8 +27,7 @@ AGENT_START = ['service', 'walinuxagent', 'start']
 AGENT_START_BUILTIN = "__builtin__"
 BOUNCE_COMMAND = [
     'sh', '-xc',
-    "i=$interface; DHCP_HOSTNAME=$hostname; x=0; "
-    "ifdown $i || x=$?; ifup $i || x=$?; exit $x"
+    "i=$interface; x=0; ifdown $i || x=$?; ifup $i || x=$?; exit $x"
 ]
 # azure systems will always have a resource disk, and 66-azure-ephemeral.rules
 # ensures that it gets linked to this path.
-- 
1.8.3.1

